mobile/Yote/js/global/utils/notificationUtils.js

```diff
@@ -0,0 +1,46 @@
+const request = require(‘request’); 
+
+const notificationUtils = {
+  sendPush(receiver, text, callback) {
+      console.log("sendPush fired..."); 
+      // Check params
+      // This is the notification payload, this can vary based on what type of notification you want to send
+      var data = JSON.stringify({
+          'to': receiver
+          , 'notification': {
+              'title': 'Yote'
+              , 'text': text
+              , 'badge': 1
+          }
+          , 'priority': 'high'
+        });
+      request(
+      {
+        url: 'https://gcm-http.googleapis.com/gcm/send' // This might be outdated - firebase url did not work at first 
+        , method: 'POST'
+        , headers: {
+          'Content-Type': 'application/json'
+          , 'Authorization': `key=xxxxxxxx` // This is the server  API key, save this in secrets
+        }
+        ,  body: data
+      }, function(error, res, body){ 
```

Comment:
use arrow functions. also, this seems like it should live in the server folder and not on the mobile side, is that correct? assuming that the server would hit this when something triggers a send